crates/extensions/tedge_flows/src/transformers/mod.rs

```diff
@@ -0,0 +1,159 @@
+use crate::js_value::JsonValue;
+use crate::FlowError;
+use crate::LoadError;
+use crate::Message;
+use std::collections::HashMap;
+use std::time::SystemTime;
+
+mod add_timestamp;
+mod cap_payload_size;
+mod set_topic;
+
+pub trait Transformer: Send + Sync + 'static {
```

Comment:
```suggestion
pub trait BuiltinStep: Send + Sync + 'static {
```
Just to avoid introducing yet another name/concept for the user, if the answer to a user asking "what is a transformer", is gonna be "it's a built-in step".